packages/terra-date-input/translations/sv.json

```diff
@@ -1,24 +1,29 @@
 {
-  "Terra.date.input.optional": "(valfri)",
-  "Terra.date.input.required": "Obligatorisk",
-  "Terra.date.input.monthLabel": "Månad",
+  "Terra.date.input.april": "April",
+  "Terra.date.input.august": "Augusti",
+  "Terra.date.input.dayDescription": "Ett tvåsiffrigt tal",
+  "Terra.date.input.dayInputLabel": "Dag",
+  "Terra.date.input.dayInputLabelWithDefaultName": "Dag i datum",
+  "Terra.date.input.dayInputLabelWithName": "Dag i {a11yLabel}",
+  "Terra.date.input.december": "December",
+  "Terra.date.input.february": "Februari",
+  "Terra.date.input.formatDayMonthYear": "(DD månad ÅÅÅÅ)",
+  "Terra.date.input.formatMonthDayYear": "(månad DD ÅÅÅÅ)",
```

Comment:
The month is capitalized for all other locals and the M in Månad for SV was capitalized before. It is made lower case intentionally?

Reply:
I asked this question to the Swedish translator.

Reply:
My guess is that Month used to be capitalized only because of old line 4 it was the only word and so got a Title Case capitalization. Now that the translation strings are phrases perhaps the new capitalization is more normal to a native speaker. I'll update the PR if they say it's wrong.